src/ui/templates/components/date-input.njk

```diff
@@ -17,7 +17,7 @@
     namePrefix: fieldId,
     fieldset: {
       legend: {
-        html: "<span class='govuk-fieldset__legend govuk-!-font-weight-bold govuk-!-font-size-24' data-cy='" + fieldId + "-label'>" + label + "</span>"
+        html: "<span class='govuk-fieldset__legend govuk-!-font-weight-bold govuk-!-font-size-24' id='" + fieldId + "-label' data-cy='" + fieldId + "-label'>" + label + "</span>"
```

Comment:
is it worth putting everything before > on a new line? for readability

Reply:
Great shout